core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala

```diff
@@ -119,6 +119,7 @@ class ResetConsumerGroupOffsetTest extends KafkaServerTestHarness {
     printConsumerGroup()
 
     AdminUtils.deleteTopic(zkUtils, topic1)
+    consumerGroupCommand1.close()
```

Comment:
It looks like `new ConsumerGroupExecutor(brokerList, 1, group, topic1)` is not shutdown. Would it make sense to have a factory method that ensures that everything is shutdown in a tearDown method like we do elsewhere?